app/models/state_file_az_intake.rb

```diff
@@ -49,6 +53,8 @@ class StateFileAzIntake < StateFileBaseIntake
   enum charitable_contributions: { unfilled: 0, yes: 1, no: 2 }, _prefix: :charitable_contributions
   enum eligibility_married_filing_separately: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_married_filing_separately
   enum eligibility_529_for_non_qual_expense: { unfilled: 0, yes: 1, no: 2 }, _prefix: :eligibility_529_for_non_qual_expense
+  enum primary_esigned: { unfilled: 0, yes: 1, no: 2 }, _prefix: :primary_esigned
+  enum spouse_esigned: { unfilled: 0, yes: 1, no: 2 }, _prefix: :spouse_esigned
```

Comment:
Even though this is stored as an enum, it doesn't look like there is any way to input a "no" option. How have we done other signatures? Are they booleans? Just the date? I think if we were trying to store only the necessary data, a primary/spouse_esigned_at field would be sufficient, though we might have to adjust the form & template to set the timestamp based on the checkbox value.

Reply:
@bengolder I based this off the `consented_to_legal` field on the CTC intake which also did a unfilled/yes/no pattern and a matching consented_to_legal_at date field. I agree that I think a boolean would also make sense or just use the date field but I like keeping in line with what we've done in the past more for consistency.